backend/chalice/api_server/run_dev_server.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
```

Comment:
how is this different from https://github.com/chanzuckerberg/corpora-data-portal/blob/main/backend/chalice/api_server/run_local_server.py If this is a replacement then please remove or update the previous.

Reply:
fair enough. Updated!